rpm/touchegg.spec

```diff
@@ -18,7 +23,7 @@ For example, you can swipe up with 3 fingers to maximize a window or swipe left
 
 
 %prep
-%setup -n touchegg
+%setup
```

Comment:
Why is the `name` option not working? I wasn't able to run it on Fedora without this flag, did I miss something in my command?

Reply:
This is because by default the `%setup` macro expects a `%{name}-%{version}` subdir where the sources are unpacked from the tarball. The `name` option was needed because your tarball was just using the `%{name}` as subdir. This is related to the need of the transform while manually creating the rpm.